src/Microsoft.DotNet.Build.Tasks/CleanupVSTSAgent.cs

```diff
@@ -2,10 +2,48 @@ namespace Microsoft.DotNet.Build.Tasks
 {
     public class CleanupVSTSAgent : BuildTask
     {
+        public bool Clean { get; set; }	
+	
+        public bool Report { get; set; }
+
+        public string AgentDirectory { get; set; }	
```

Comment:
I would keep the Required attribute on AgentDirectory and RetenionDays.

Reply:
Done. Going up on next commit.